tcks/profiles/platform/signaturevalidation/generate/pom.xml

```diff
@@ -0,0 +1,829 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Generate Web Profile + Platform TCK signature map files for the following:
+jakarta.annotation.sig
+jakarta.batch.sig
+jakarta.decorator.sig
+jakarta.ejb.sig
+jakarta.el.sig
+jakarta.enterprise.concurrent.sig
+jakarta.enterprise.sig (CDI)
+jakarta.faces.sig
+jakarta.inject.sig
+jakarta.interceptor.sig
+jakarta.jms.sig
+jakarta.json.bind.sig
+jakarta.json.sig (JSON-P)
+jakarta.mail.sig
+jakarta.persistence.sig
+jakarta.resource.sig
+jakarta.security.auth.message.sig
+jakarta.security.enterprise.sig
+jakarta.security.jacc.sig
+jakarta.servlet.jsp.jstl.sig (Tags)
+jakarta.servlet.jsp.sig
+jakarta.servlet.sig
+jakarta.transaction.sig
+jakarta.validation.sig
+jakarta.websocket.sig
+jakarta.ws.rs.sig
+javax.rmi.sig (Corba classes for EJB)
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.9</version>
+    </parent>
+
+    <artifactId>generate</artifactId>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
If we're not assembling these in a JAR these could be removed and we could make this a `<packaging>pom</packaging>`. If we do want a JAR, we should instead use the `maven.compiler.release` property.